lib/elixir/lib/enum.ex

```diff
@@ -2249,11 +2249,16 @@ defmodule Enum do
 
   """
   @spec take_random(t, non_neg_integer) :: list
+  def take_random(enumerable, count)
+
   def take_random(_enumerable, 0), do: []
-  def take_random(first..first, 1),
+
+  def take_random(first..first, count) when is_integer(count) and count >= 1,
     do: [first]
+
   def take_random(first..last, 1) when first > last,
     do: take_random(last..first, 1)
+
```

Comment:
true. fixed, i just split https://github.com/elixir-lang/elixir/pull/4869/files#diff-6881431a92cd4e3ea0de82bf2338f8eaR2254 to make it consistent